packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -123,114 +124,119 @@ contract DeploySuperchainInput is CommonBase {
     // fields. With getter methods, we can be sure that the input is set before accessing any field.
 
     function proxyAdminOwner() public view returns (address) {
-        assertInputSet();
-        return inputs.roles.proxyAdminOwner;
+        require(_proxyAdminOwner != address(0), "DeploySuperchainInput: proxyAdminOwner not set");
+        return _proxyAdminOwner;
     }
 
     function protocolVersionsOwner() public view returns (address) {
-        assertInputSet();
-        return inputs.roles.protocolVersionsOwner;
+        require(_protocolVersionsOwner != address(0), "DeploySuperchainInput: protocolVersionsOwner not set");
+        return _protocolVersionsOwner;
     }
 
     function guardian() public view returns (address) {
-        assertInputSet();
-        return inputs.roles.guardian;
+        require(_guardian != address(0), "DeploySuperchainInput: guardian not set");
+        return _guardian;
```

Comment:
Losing the roles prefix from the struct here feels like a bit of a loss to clarity/organization.

Have you considered namespacing them with a simple prefix, like _rolesGuardian, or maybe _roles_guardian?

Reply:
Yea, I also didn't love that loss of organization

One thing that always confused me in our original `Deploy.s.sol` script is when things don't match (e.g. `finalSystemOwner` in deploy config, but that's not the name of a role). So "continuity" was a principal I tried to follow, e.g. the name used in an input file (like `guardian`) should match the getter name in the input contract, and that should match the getter name in the SuperchainConfig. That led me to not use any namespacing like this

Reply:
Ok great, I've added a sticky to the figjam as well for this to ensure we track it